.github/workflows/ci.yaml

```
@@ -0,0 +1,33 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request: null # target every PR
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: earthly/actions/setup-earthly@v1
+        with:
+          version: v0.6.30
```

```suggestion
      - uses: earthly/actions-setup@v1
        with:
          version: v0.7.15
```
The `actions` repo has been moved to `actions-setup` 
https://github.com/earthly/actions

Also think, we can use the latest earthly version

---

Ack, I'll do this and the v3 of the checkout action..

I think I'll also now raise a parallel PR against https://github.com/expressvpn/wolfssl-sys to refresh that too

---

Parallel PR: https://github.com/expressvpn/wolfssl-sys/pull/38